config/redisConfig.js

```diff
@@ -29,6 +29,12 @@ module.exports = {
     cache: pe.REDIS_CACHE && pe[pe.REDIS_CACHE] ?
       pe[pe.REDIS_CACHE] : PRIMARY_REDIS,
 
+    /*
+     * Track generator changes to be sent in collector heartbeat response.
+     */
+    heartbeat: pe.REDIS_HEARTBEAT && pe[pe.REDIS_HEARTBEAT] ?
+      pe[pe.REDIS_HEARTBEAT] : PRIMARY_REDIS,
+
```

Comment:
Do we want to use a separate instance just to track the collectors?

Reply:
We don't need to provision a separate instance for it, but I think it makes sense to allow an admin to assign it where they want.